src/Jackett.Common/Definitions/xbytes2.yml

```diff
@@ -114,6 +121,12 @@ search:
     title:
       selector: td[valign="middle"] a
       filters:
+        - name: re_replace
+          args: ["\\W", "."] # Spaces and other characters -> .
```

Comment:
Replacing spaces with dots is not necessary. Sonarr/Radarr works well with spaces and the text is more readable.

Reply:
Removed.

The another two dots related replace fix issues on naming.